src/elm/View/Section1.elm

```diff
@@ -3,13 +3,18 @@ module View.Section1 exposing (view)
 import Data
 import Html
 import Model exposing (Model)
-import Msg exposing (Msg)
+import Msg exposing (Msg(..))
+import Pile
 import View.MainText
 import View.Posts
 
 
 view : Model -> List (Html.Html Msg)
 view model =
     [ View.MainText.viewTop Data.Section1 model.content.mainText
-    , View.Posts.view Data.Section1 model.content.posts
+    , if Tuple.second model.viewportHeightWidth < 800 then
+        View.Posts.view Data.Section1 model.content.posts
+
+      else
+        Pile.view model.pile1.pile |> Html.map Pile1
```

Comment:
Is this because `Post`s as well as `StackingImage`s can be `Pile`s? I may need to adjust my thinking on the data modelling if that is the case.